coalib/bearlib/aspects/Smell.py

```diff
@@ -43,21 +47,23 @@ class docs:
 @Smell.subaspect
 class ClassSmell:
     """
-    This aspect detects `code smells` or `bad smells` related to classes'
-    definitions in your codebase.
+    Code smells related to classes' definition.
 
     Class-level code smells indicate poorly defined classes (including too
     large classes or God object, data clump feature envy etc...) in your
     source code.
     """
     class docs:
         example = """
-        * Too large classes
-        * Data clump
-        * Feature envy
-        etc ...
+        class Warehouse
+            def sale_price(item)
+                (item.price - item.rebate) * @vat
```

Comment:
instance variable `@vat` is never set.
This example would be better without any use of instance variables.